docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -81,8 +81,10 @@ Server ID                            Hostname        Services Version Upgrader
 
 </Details>
 
-<Tabs dropdownCaption="Cluster Type" dropdownSelected="Self-hosted">
-<TabItem label="systemd" options="Self-hosted">
+<Tabs dropdownView dropdownCaption="Cluster Type">
```

Comment:
Since there's no introductory text before the inner `TabItem`s, I think it would make sense to remove the `dropdownView` and use `options` to configure the dropdown menu. Here are the instructions for this:

https://goteleport.com/docs/contributing/documentation/reference/#two-dimensional-tabs-components